src/behaviors/Behavior.js

```diff
@@ -162,16 +236,18 @@ var Behavior = Base.extend('Behavior', {
     },
 
     clearColumns: function() {
-        var schema = this.dataModel.schema,
+        var schema = this.schema,
             tc = this.treeColumnIndex,
             rc = this.rowColumnIndex;
 
         schema[tc] = schema[tc] || {
```

Comment:
Per @nmichaud: App-provided schema should be able to specify name and header of tree column (as it can any other column). Therefore, don't override with default tree column element when schema already defines.